front/app/components/UI/QuillEditor/altTextToImagesModule.tsx

```diff
@@ -14,24 +14,20 @@ export class ImageBlot extends BlockEmbed {
   static blotName = 'image';
   // Instead of using the default img tag, we are using a div where we can add an input field for the alt text
   static tagName = ['div'];
+  // We are setting a custom class on the div so that we can reference it later on
+  static className = 'ql-alt-text-input-container';
```

Comment:
Apparently, it's important to add a class name in this way for Quill to be able to differentiate between registered blots.